Formula/fdroidserver.rb

```diff
@@ -15,25 +15,35 @@ class Fdroidserver < Formula
 
   depends_on :python3
   depends_on "freetype"
-  depends_on "jpeg"
```

Comment:
I'm not completely sure to be honest. `pillow` blew up without `openjpeg` and it seemed weird that it would want both `jpeg` and `openjpeg` so I tried dropping the former and it built & tested fine.